src/lips.cc

```diff
@@ -261,7 +264,7 @@ int lipsLoad(const char* audioFileName, const char* headFileName)
         *sep = '\0';
     }
 
-    strcpy(gLipsData.field_50, v60);
+    strncpy(gLipsData.field_50, v60, std::min(strlen(v60), sizeof(gLipsData.field_50)));
```

Comment:
Is strlen here necessary? From docs, strncpy will pad dest with zeroes up to a given length.

Reply:
You're right, the extra code isn't really needed here. Fixed!